app/controllers/nonsipp/memberdetails/CheckMemberDetailsFileController.scala

```diff
@@ -158,14 +171,44 @@ class CheckMemberDetailsFileController @Inject()(
     duration
   )
 
-  private def auditDownload(srn: Srn, responseStatus: Int, startTime: Long)(implicit request: DataRequest[_]): Unit = {
-    val endTime = System.currentTimeMillis
-    val duration = endTime - startTime
+  private def auditDownload(srn: Srn, responseStatus: Int, duration: Long)(implicit request: DataRequest[_]): Unit =
```

Comment:
it seems auditDownload/auditValidation method signature says return type unit but we are returning something else. It's not a big deal but a bit confusing

Reply:
Will push the changes as a separate branch @cyeniceri as this was merged.

Reply:
thanks @kberec